lib/omnibus-ctl.rb

```diff
@@ -506,6 +508,45 @@ def reconfigure(exit_on_success=true)
       end
     end
 
+    def check_license_acceptance(override_accept = false)
+      license_guard_file_path = File.join(etc_path, ".lic.accepted")
+
+      # If the project does not have a license we do not have
+      # any license to accept.
+      return unless File.exist?(project_license_path)
+
+      if !File.exist?(license_guard_file_path)
+        if override_accept || ask_license_acceptance
+          FileUtils.touch(license_guard_file_path)
+        else
+          puts "Please accept software's license in order to continue."
+          exit! 1
+        end
+      end
+    end
+
+    def ask_license_acceptance
+      system("less #{project_license_path}")
```

Comment:
How is the user going to know what is happening at this point? How will they know how to quit less?